apps/vaporgui/QRangeSlider.h

```diff
@@ -35,6 +35,8 @@ class QRangeSlider : public QSlider {
     
     int _position[2];
     int _value[2];
+    bool _isOutOfBounds[2] = {false};
```

Comment:
Can you initialize an array of size two with a single elements?

Reply:
Yes